plugin/pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -92,6 +92,8 @@ func TestAssumePodScheduled(t *testing.T) {
 			}
 		}
 		n := cache.nodes[nodeName]
+		// Ignore generation field.
```

Comment:
can you add a little helper function for node info comparison instead of clearing the generation filed before every deepequal?